data/bestiary/creatures-aoa3.json

```diff
@@ -3537,7 +3537,7 @@
 					{
 						"amount": 8,
 						"name": "all damage",
-						"note": "(except force, {@item ghost touch}, or positive; double resistance vs. non-magical)"
+						"note": "(except force, ghost touch, or positive; double resistance vs. non-magical)"
```

Comment:
Why remove the ghost touch tag?

Reply:
I was fixing a couple of these notes which occur quite frequently and almost none had this reference and item seemed to be wrong at first glance so i normalized it to the more common version.

But since there isn't a general ghost touch trait, I guess linking to the rune here isn't a bad idea maybe. Although in that case we should probably adjust all occurrences of this note so it is consistent.

Reply:
Yeah, ghost touch in this situation specifically refers to the rune, so it's best to tag it in every instance.

Reply:
Added a commit that links all the various ghost touch references accordingly.